impeller/entity/shaders/glyph_atlas.frag

```diff
@@ -16,5 +16,5 @@ out f16vec4 frag_color;
 
 void main() {
   f16vec4 value = texture(glyph_atlas_sampler, v_uv);
-  frag_color = value.aaaa * v_text_color;
+  frag_color = value.rrrr * v_text_color;
```

Comment:
I suggested a non-swizzling texture read with an ifdef yesterday, but later realized you could also just do `vec4(max(value.a, value.r))` instead. I believe those are even branchless.

Reply:
This does not work, as when binding to an R8 texture the alpha channel is 1.0. I'm not sure if this is specified behavior or not. We can use a specialization constant to handle this.